infra/README.md

```diff
@@ -0,0 +1,10 @@
+# Deploy
+
+You'll need to be sure to get setup with you `tfvars` from a friendly developer near you.
```

Comment:
Not sure I spotted the typo but I changed it as it was a rubbish sentence anyway.